### DISCUSSION TITLE
Save the best model

### DISCUSSION THREAD

**Discussion by oguzhankir** [yandex-research/rtdl-revisiting-models#18] [Q&A] [answered]
How do I save the model with the best validation score among the epochs?

**Comment by Yura52** [comment, marked as answer]
The best model is saved as a part of the checkpoint. For example, for MLP, see these lines:
- https://github.com/Yura52/tabular-dl-revisiting-models/blob/c7227f0f091e5dfdd4df2c3712307a3a06ac32c2/bin/mlp.py#L139
- https://github.com/Yura52/tabular-dl-revisiting-models/blob/c7227f0f091e5dfdd4df2c3712307a3a06ac32c2/bin/mlp.py#L201
- https://github.com/Yura52/tabular-dl-revisiting-models/blob/c7227f0f091e5dfdd4df2c3712307a3a06ac32c2/bin/mlp.py#L256
- https://github.com/Yura52/tabular-dl-revisiting-models/blob/c7227f0f091e5dfdd4df2c3712307a3a06ac32c2/bin/mlp.py#L271
